src/ai/susi/server/api/cms/FiveStarRateSkillService.java

```diff
@@ -184,6 +184,7 @@ public JSONObject createRatingObject(String skill_stars) {
 
         float totalStars = skillStars.getInt("one_star") + skillStars.getInt("two_star") + skillStars.getInt("three_star") + skillStars.getInt("four_star") + skillStars.getInt("five_star");
         float avgStar = (1 * skillStars.getInt("one_star") + 2 * skillStars.getInt("two_star") + 3 * skillStars.getInt("three_star") + 4 * skillStars.getInt("four_star") + 5 * skillStars.getInt("five_star")) / totalStars;
+        avgStar = Math.round(avgStar*100.0)/100.0;
```

Comment:
Something about this calculation seems to be off: Here's an example:

Let us consider the star ad it's equivalent count:
```
1 star => 5
2 star => 10
3 star => 5
4 star => 3
5 star => 40
```
Therefore `totalStars` contains `5+10+5+3+40 = 63`
and `avgStar` in line `186` now contains `(5*1 + 10*2 + 5*3 + 3*4 + 40*5)/63 = 252/63` which results in `4.0`
Why do you have to do another `avgStar * 100.0/100.0` here, it doesn't seem to be the right thing to do even though the number is the same. You could directly use the `avgStar` value you computed and avoid this computation.

Reply:
@sudheesh001 This line was written to keep only 2 places after decimal. However, I shall remove this line for now and open a new PR for the issue.

Reply:
You're anyway using `float` to store the value of the average, if you'd like only 2 decimal places after that, run `Math.round()` on that line itself which'll convert the value as you need it to the number of decimal places.

Reply:
@sudheesh001 I have sent #801 for the fix and have removed those changes from this PR.